sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/SonarScanner.java

```diff
@@ -195,6 +197,36 @@ public boolean isUseOldSonarRunnerScript() {
     return !scannerVersion().isGreaterThanOrEquals("2.6") || useOldSonarRunnerScript;
   }
 
+  /**
+   * @since 3.15 used by SQ Scanner CLI ITs
+   */
+  public SonarScanner useNative(boolean useNative) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Provide multiple methods instead of using "useNative" to determine which action to take. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2301)